src/gevent/greenlet.py

```diff
@@ -521,6 +529,41 @@ def start_later(self, seconds):
             self._start_event = self.parent.loop.timer(seconds)
             self._start_event.start(self.switch)
 
+    @staticmethod
+    def add_spawn_callback(cb):
+        """
+        add_spawn_callback(callback) -> None
+
+        Set up a *callback* to be invoked when a new :class:`Greenlet`
+        object is instantiated.
+
+        The invocation order of spawn callbacks is unspecified.  Adding one
```

Comment:
"Adding a callback more than once ..." might be even clearer here. What do you think?